doc/ref/reesmat.xml

```diff
@@ -360,16 +360,20 @@ false]]></Example>
 </ManSection>
 
   <ManSection>
-    <Attr Name="Matrix" Arg="R"/>
+    <Oper Name="Matrix" Arg="R"/>
```

Comment:
As soon as the operation `Matrix` becomes documented in the MatrixObj context,
we will need labels to distinguish the different declarations.
We can add the `Label="for Rees matrix semigroups"` attribute already now,
and use it in the `<Ref>` elements below.

Reply:
Thanks, this will be fixed in the next update of this PR.